UnicornRender/video/include/unicorn/video/Primitives.hpp

```diff
@@ -24,15 +27,15 @@ class Primitives
     *  @param[out] mesh    geometry mesh
     *  @return reference to updated mesh
     */
-    UNICORN_EXPORT static Mesh& Box(Mesh& mesh);
+    UNICORN_EXPORT static Mesh* Box();
```

Comment:
Documentation appears to be incorrect now. This is now always _creates_ a mesh? It is not possible to fill existing mesh with geometry data anymore?